### ISSUE TITLE
Previous days and monthly data

### ISSUE THREAD

**Issue by lampra1** [philipptrenz/sunportal#1]
Hi,
I was searching for such an offline visualization tool. This seems as a good start. Thank you
I have installed it on cubietruck running armbian (debian).
Two questions:
1. Is it supposed to present daily plots for previous days or just the day total? On my installation, only the day total is presented but no plot for previous days.
2. Are you planning to provide monthly plots?
Thank you

**Comment by philipptrenz** [comment]
Hi @lampra1, good to hear that it is useful to you!

1. On site load it shows the current day plot, but if there is data for previous days available a navigation button shows up and you can navigate to the day before (and back). The first container with today's and total yield and CO2 savings is always up to date with the latest data of today.
1. Yes, I just finished this feature, it's a bar plot underneath the day plot. I still have to test it, but give it a try ;)

**Comment by lampra1** [comment]
Hi @philipptrenz 
Thank you for your prompt reply.
I installed last night and tested again today. It seems that it cant handle right now multiple inverters as today it does not show the day plot but only the total. Is it supposed to work with one inverter only at the moment? 
I will test the monthly feature later today.

**Comment by philipptrenz** [comment]
Hm, I have it running with two inverters and it sums up the two just fine. Did you ensure that SBFspot retrieves the data of all your inverters?

If you send me your db-file per mail I‘ll check what the problem is about.

**Comment by lampra1** [comment]
Hi, I was busy. Is this the way to send you the email? or you need to give me your email address?
    Στις Κυριακή, 9 Σεπτεμβρίου 2018, 11:58:06 μ.μ. EEST, ο χρήστης Philipp Trenz <notifications@github.com> έγραψε:  
 
 Hm, I have it running with two inverters and it sums up the two just fine. Did you ensure that SBFspot retrieves the data of all your inverters?

If you send me your db-file per mail I‘ll check what the problem is about.

**Comment by philipptrenz** [comment]
Please send to mail@philipptrenz.de

**Comment by philipptrenz** [comment]
Hi @lampra1,

I tested with your data and found the problem, just reinstall _sunportal_ and it should work.

I also saw a wrong negative yield value in the month plot at August 6, 2018. This seems to be an issue of SBFspot itself, the day yield of your inverter with the serial number ending with 28 has a negative value at this date inside the table MonthData. You might create an issue for that at https://github.com/SBFspot/SBFspot.

Greetings
